vmm/src/lib.rs

```diff
@@ -1298,6 +1309,8 @@ impl RequestHandler for Vmm {
 
     fn vm_snapshot(&mut self, destination_url: &str) -> result::Result<(), VmError> {
         if let Some(ref mut vm) = self.vm {
+            //drain console_info so that FDs are not reused
```

Comment:
Missing a space after `//` .

The comment style in code is to properly capitalize a sentence. Please fix all the comments.